presto-tests/src/main/java/com/facebook/presto/tests/DistributedQueryRunner.java

```diff
@@ -335,13 +342,44 @@ public void createCatalog(String catalogName, String connectorName, Map<String,
         log.info("Announced catalog %s (%s) in %s", catalogName, connectorId, nanosSince(start));
     }
 
+    @Override
     public void loadFunctionNamespaceManager(String functionNamespaceManagerName, String catalogName, Map<String, String> properties)
     {
         for (TestingPrestoServer server : servers) {
             server.getMetadata().getFunctionManager().loadFunctionNamespaceManager(functionNamespaceManagerName, catalogName, properties);
         }
     }
 
+    /**
+     * This method exists only because it is currently impossible to create a function namespace from the query engine,
```

Comment:
Why it's not possible to "create a function namespace from the query engine"? What does this mean?

Reply:
Before this change, users, clients, test with access to QueryRunner (which is essentially a client), and even the `TestingPrestoServer` does not have a way to create a function namespace after a function catalog is setup. Therefore, in order for test to create a function in `testing.test` after we load a function namespace manager for the catalog `testing`, we need a way to create `testing.test`.

Currently, we won't be able to create `testing.test` with a client, so we need to add this method to run a query against the db to create to the function namespace. If in the future, we support creating function namespace with `CREATE SCHEMA`, this method will no longer be needed, and the test runner no longer need to store the DB Handle which it should not.

Reply:
Change the comment to presto server then.